omnibus/package-scripts/chef-workstation/postinst

```diff
@@ -77,7 +77,7 @@ fi
 
 
 
-binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client hab"
+binaries="chef-run berks chef chef-cli chef-automate-collect chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client hab"
```

Comment:
```suggestion
binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client hab"
```

I propse that we don't link these onto the path because only `chef push` is meant to call that and it should be able to figure out the `embedded/bin` folder

Reply:
Latest tech design discussions in the team suggest we will not modify chef push, but instead chef cli wrapper will be modified to make a call to chef push and then to chef-automate-collect. We can discuss if that is not the case.

Reply:
Even if we are calling `chef-automate-collect` from the `chef` wrapper, we can still reference it from `/opt/chef-workstation/embedded/bin` instead of `/usr/local/bin` (which is where these symlinks end up).

I don't want to hold up this PR anymore but I would like to reconsider these symlinks when we wire up the `chef` wrapper to call `chef-automate-collect`